app/assets/javascripts/rails_admin/ra.widgets.coffee

```diff
@@ -46,7 +58,7 @@ $(document).on 'rails_admin.dom_ready', (e, content) ->
       input = this
       image_container = $("#" + input.id).parent().children(".preview")
       unless image_container.length
-        image_container = $("#" + input.id).parent().prepend($('<img />').addClass('preview')).find('img.preview')
+        image_container = $("#" + input.id).parent().prepend($('<img />').addClass('preview').addClass('img-thumbnail')).find('img.preview')
```

Comment:
You don't need to chain `addClass`, you can do `addClass('foo bar')`

https://api.jquery.com/addclass/